HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -7070,6 +7186,53 @@ def initialize(hpxml_element, *args, **kwargs)
     attr_reader(*CLASS_ATTRS)
     attr_accessor(*ATTRS)
 
+    # Returns whether, based on compressor/backup heating lockout temperatures, compressor and backup system can both operate at the same time.
+    #
+    # @return [Boolean] True if backup system can operate at the same time as the compressor
+    def simultaneous_backup
+      if !@compressor_lockout_temp.nil? &&
+         !@backup_heating_lockout_temp.nil? &&
+         (@backup_heating_lockout_temp > @compressor_lockout_temp)
+        return true
+      end
+
+      return false
+    end
```

Comment:
I feel like there is a misunderstanding here. They won't operate at the same time, they are just both available between those two temperatures.

Reply:
I'll remove this new method.

Reply:
I wish I knew a long time ago that HP and backup can never operate simultaneously (even within a switchover band). In terms of a power rating for heating, this simplifies things A LOT.

Reply:
Hmm, sorry, I may need to walk this back a bit. I was mostly thinking about dual-fuel HPs (HPs with integrated non-electric backup). For your purpose, I guess you are really focused on HPs with integrated electric backup. Let me double-check that...

Reply:
Alright, it does seem like integrated electric backup can run simultaneously with the HP. Sorry, I think I was thrown off by the method name (`simultaneous_backup`). It _presumes_ there is simultaneous operation just because there is a gap between the compressor lockout temp and backup lockout temp, but that's not right. In some cases (like a DFHP), they typically cycle instead.

So a better name would be something like... `overlapping_compressor_and_backup_operation`? And add a comment that in this temperature range there may be simultaneous operation or cycling.

Reply:
I made the name change, and added a comment.

Reply:
https://github.com/NREL/OpenStudio-HPXML/pull/1946/commits/244dcc2c5d0abe1fac58fcf26f0545e4bac995cf